.gitignore

```diff
@@ -1 +1,2 @@
-deps/
\ No newline at end of file
+deps/
+.eca/
```

Comment:
Curious why excluding a local eca folder, I believe most of config could just go at `~/.config/eca/config.json`?

Reply:
I use a local `.eca` configuration file